idaes/core/surrogate/alamopy.py

```diff
@@ -1226,25 +1226,43 @@ def evaluate_surrogate(self, inputs):
 
     def populate_block(self, block, additional_options=None):
         """
-        Method to populate a Pyomo Block with surrogate model constraints.
+        Method to populate a Pyomo Block with surrogate model constraints
+        or expressions.
 
         Args:
             block: Pyomo Block component to be populated with constraints.
-            additional_options: None
-               No additional options are required for this surrogate object
+            additional_options (dict): as_expression (bool) is an optional
```

Comment:
1. Doc string does not match argument.
2. Why make this a dict of options rather than a list of separate arguments?

Reply:
I didn't write it, it was already there and already a dict.  I think the dict comes from keyword args to ``build_model`` in the general class.  This isn't something users use directly.  I'll fix the doc string, but I'm not really sure how to properly document it.

I suppose the right way would be to add it to the build_model doc string, but it only works with ALAMO for now.

Reply:
Is this method called indirectly via another method on the actual SurrogateModel block (i.e. `build_model`)?